solidity-plugin/src/main/java/org/sonarsource/solidity/SyntaxHighlightingVisitor.java

```diff
@@ -24,10 +24,10 @@ public void visitTokens(TokenStream tokenStream) {
       } else if (SoliditySensor.KEYWORD_TYPES.contains(token.getText())) {
         this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
           token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.KEYWORD_LIGHT);
-      } else if (token.getType() == 115) {
+      } else if (SolidityParser.typeMatches(token, SolidityParser.StringLiteral)) {
         this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
           token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.STRING);
-      } else if (token.getType() >= 96 && token.getType() <= 100) {
+      } else if (SolidityParser.typeMatches(token, 95, 96, 97, 98, 99, 100)) {
         this.highlighting.highlight(token.getLine(), token.getCharPositionInLine(),
           token.getLine(), (token.getCharPositionInLine() + token.getStopIndex() - token.getStartIndex() + 1), TypeOfText.CONSTANT);
       } else {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([squid:S00108](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS00108))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=19&issues=AWQYmPFq_bEzQ-kUQY4a&open=AWQYmPFq_bEzQ-kUQY4a)

[ISSUE_KEY]: AWQYmPFq_bEzQ-kUQY4a